wp-content/plugins/core/src/Blocks/Blocks_Definer.php

```diff
@@ -30,6 +30,7 @@ class Blocks_Definer implements Definer_Interface {
 	public function define(): array {
 		return [
 			self::TYPES => DI\add( [
+				DI\get( Types\Gallery_Grid\Gallery_Grid::class ),
```

Comment:
Would you kindly import (`use ...`) the namespace and then class the class name only here?